plugin/storage/cassandra/spanstore/dbmodel/tag_filter_blacklist.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
my directory search doesn't show you working for Uber :-) . This line is not needed.

Reply:
Ha.  Nope. :)